agent/src/agent/pipeline/manager.py

```diff
@@ -327,13 +330,23 @@ def get_sample_records(pipeline_: Pipeline) -> (list, list):
 
 
 def _get_preview_data(test_pipeline: Pipeline):
-    streamsets.manager.create(test_pipeline)
+    sdc_client.create(test_pipeline)
     try:
-        preview = streamsets.manager.create_preview(test_pipeline)
-        preview_data, errors = streamsets.manager.wait_for_preview(test_pipeline, preview['previewerId'])
+        preview = sdc_client.create_preview(test_pipeline)
+        preview_data, errors = sdc_client.wait_for_preview(test_pipeline, preview['previewerId'])
     except (Exception, KeyboardInterrupt) as e:
         logger_.exception(str(e))
         raise
     finally:
-        streamsets.manager.delete(test_pipeline)
+        sdc_client.delete(test_pipeline)
     return preview_data, errors
+
+
+def create_streamsets_pipeline_config(pipeline_: Pipeline) -> dict:
+    return get_config_handler(pipeline_).override_base_config()
+
+
+def _get_config_loader(pipeline_: Pipeline):
```

Comment:
Don't you have this method in agent/src/agent/pipeline/config/handlers/factory.py?